frontend/src/pages/modelServing/screens/global/ModelServingGlobal.stories.tsx

```diff
@@ -143,3 +143,58 @@ DeleteModel.play = async ({ canvasElement }) => {
   await userEvent.type(retypeNameInput, 'Test Inference Service', { delay: 50 });
   expect(deleteButton).not.toBeDisabled();
 };
+
+export const DeployModel = Template.bind({});
+DeployModel.parameters = {
+  a11y: {
+    // need to select modal as root
+    element: '.pf-c-backdrop',
+  },
+};
+DeployModel.play = async ({ canvasElement }) => {
+  // load page and wait until settled
+  const canvas = within(canvasElement);
+  await canvas.findByText('Test Inference Service', undefined, { timeout: 5000 });
+
+  // user flow for editing a project
+  await userEvent.click(canvas.getByText('Deploy model', { selector: 'button' }));
```

Comment:
Going forward I think we might want to look at adding test ids -- selecting on text is not the best.